extensions/amp-ima-video/0.1/amp-ima-video.js

```diff
@@ -0,0 +1,201 @@
+import {getIframe, preloadBootstrap} from '../../../src/3p-frame';
```

Comment:
Let's put an experimental flag around this component and do an experimental launch with page-level opt-in support (so publishers can actually ship with it)

I created https://github.com/ampproject/amphtml/projects/24 to track issues we need to address before a GA launch.

To add experimental flag:
1- Add an entry to `experiments.js` file
2- As first line of `buildCallback` do `user().assert(isExperimentOn(this.win, TAG),
        `Experiment "${TAG}" is disabled.`);`
3- Add TAG to `allow-doc-opt-in` in prod-config.json and canary-config.json (must be a separate PR after this is merged)